src/os_posix/os_fs.c

```diff
@@ -796,8 +796,9 @@ __posix_file_write_mmap(
         /* If we are actively extending the file, don't remap it on every write. */
         if ((remap_opportunities++) % WT_REMAP_SKIP == 0) {
             __wti_posix_prepare_remap_resize_file(
-              file_handle, wt_session, offset + (wt_off_t)len, NULL);
-            __wti_posix_remap_resize_file(file_handle, wt_session);
+              file_handle, wt_session, offset + (wt_off_t)len, &remap);
+            if (remap)
```

Comment:
Similar question, is this a bugfix to some existing code?

Reply:
This is a bug fix I pulled from develop.